### ISSUE TITLE
MiniProfiler conflicts with jquery.tmpl, causing a TypeError: x.tmpl is not found

### ISSUE THREAD

**Issue by GoogleCodeExporter** [akyazilim/mvc-mini-profiler#40]
```
I saw an earlier issue where this might have been fixed, but the current 
version still causes a conflict. If I take out MiniProfiler, I do not get an 
error.

Of note, I am including jquery.tmpl v1.0.0pre via an embedded resource in my 
MVC project:

<script type="text/javascript" src="/WebResource.axd?d=xxx&amp;t=123"></script>

If that may change how you detect if it's already included.
```

Original issue reported on code.google.com by `kamran.a...@gmail.com` on 23 Jun 2011 at 6:37

**Comment by GoogleCodeExporter** [comment]
```
Are you seeing this issue when calling .RenderIncludes() *after* your current 
<script> block?  

MiniProfiler's scripts can only detect what's included before them, we run this 
check at the time of our include...which should happen after your other scripts 
so we don't override anything.
```

Original comment by `nrcraver` on 24 Jun 2011 at 10:48

**Comment by GoogleCodeExporter** [comment]
```
I did think of that after I posted but I checked and made sure to include the 
script after any page-specific scripts (where I include jquery.tmpl) and the 
problem persisted. Thanks for pointing out that is what happens, though, as I 
wasn't sure.
```

Original comment by `kamran.a...@gmail.com` on 24 Jun 2011 at 3:00

**Comment by GoogleCodeExporter** [comment]
```
You can close this; it seems to be a localized problem as it seems to work now; 
it could have been a conflict with the way we were including scripts.
```

Original comment by `kamran.a...@gmail.com` on 28 Jun 2011 at 9:00

**Comment by GoogleCodeExporter** [comment]
```
I also only pull down jquery.tmpl if it's not detected.  This functionality is 
available in the latest nuget version, 1.4.
```

Original comment by `jarrod.m.dixon` on 28 Jun 2011 at 9:56
- Changed state: **Fixed**
